tools/build_config/Jenkinsfile

```diff
@@ -342,6 +342,47 @@ pipeline {
       }
     }
 
+    stage('Configure') {
+      steps {
+        script {
+          if (env.BUILD_DOCS?.toBoolean()) {
+            if (isUnix()) {
+              sh '''
+            module load python/3.6 &&
+            pip install --user sphinx &&
+            pip install --user sphinx_rtd_theme
+            '''
+            } else {
+              powershell '''
+             pip install --user sphinx
+             pip install --user sphinx_rtd_theme
+            '''
+            }
+          }
+          if (isUnix()) {
+            sh '''
+          export PATH=${PATH}:~/.local/bin &&
```

Comment:
does Python need to be on the path here so `--print_versions` can log the Python version along with the other versions?